sopel_modules/github/github.py

```diff
@@ -426,26 +437,39 @@ def configure_repo_colors(bot, trigger):
     if len(colors) != 6:
         return bot.say('You must provide exactly 6 colors! See "{}help gh-hook-color" for more information.'.format(bot.config.core.help_prefix))
 
-    conn = bot.db.connect()
-    c = conn.cursor()
-
-    c.execute('SELECT * FROM gh_hooks WHERE channel = ? AND repo_name = ?', (channel, repo_name))
-    result = c.fetchone()
-    if not result:
-        return bot.say('Please use "{}gh-hook {} enable" before attempting to configure colors!'.format(bot.config.core.help_prefix, repo_name))
-    else:
-        combined = colors
-        combined.append(channel)
-        combined.append(repo_name)
-        c.execute('''UPDATE gh_hooks SET repo_color = ?, name_color = ?, branch_color = ?, tag_color = ?,
-                     hash_color = ?, url_color = ? WHERE channel = ? AND repo_name = ?''', combined)
-        conn.commit()
-        c.execute('SELECT * FROM gh_hooks WHERE channel = ? AND repo_name = ?', (channel, repo_name))
-        row = c.fetchone()
-        bot.say("[{}] Example name: {} tag: {} commit: {} branch: {} url: {}".format(
-                formatting.fmt_repo(repo_name, row),
-                formatting.fmt_name(trigger.nick, row),
-                formatting.fmt_tag('tag', row),
-                formatting.fmt_hash('c0mm17', row),
-                formatting.fmt_branch('master', row),
-                formatting.fmt_url('http://git.io/', row)))
+    session = bot.db.ssession()
+    try:
+        hook = session.query(GithubHooks) \
+            .filter(GithubHooks.channel == channel) \
+            .filter(GithubHooks.repo_name == repo_name) \
+            .one_or_none()
+        if hook is None:
+            return bot.say('Please use "{}gh-hook {} enable" before attempting to configure colors!'.format(bot.config.core.help_prefix, repo_name))
+        else:
+            hook.channel = channel
+            hook.repo_name = repo_name
+            hook.repo_color = colors[0]
+            hook.name_color = colors[1]
+            hook.branch_color = colors[2]
+            hook.tag_color = colors[3]
+            hook.hash_color = colors[4]
+            hook.url_color = colors[5]
+            session.commit()
+
+            row = session.query(GithubHooks) \
+               .filter(GithubHooks.channel == channel) \
+               .filter(GithubHooks.repo_name == repo_name) \
+               .one_or_none()
+
+            bot.say("[{}] Example name: {} tag: {} commit: {} branch: {} url: {}".format(
+                    formatting.fmt_repo(repo_name, row),
+                    formatting.fmt_name(trigger.nick, row),
+                    formatting.fmt_tag('tag', row),
+                    formatting.fmt_hash('c0mm17', row),
+                    formatting.fmt_branch('master', row),
+                    formatting.fmt_url('http://git.io/', row)))
```

Comment:
Just an itty-bitty thing to tweak As Long As This Line Is Touched Anyway™.

```suggestion
                    formatting.fmt_url('https://git.io/', row)))
```

Reply:
Done.